packages/sitecore-jss-rendering-host/src/devServer.ts

```diff
@@ -196,7 +196,7 @@ export function startDevServer({
     serverOptions.public = tunnelUrl;
   }
 
-  const modulePath = path.join(buildArtifactsPath, serverBundleFileName);
+  const modulePath = path.join(buildArtifactsPath, serverBundleFileName as string);
```

Comment:
I suppose that this type of change can be caused by _webpack_ upgrade, you override type but the value still can be different, need to handle a different type in this case (there can be a function, I checked)

Reply:
Yes right after the upgrade the `serverBundleFileName` also accepts a function. I have refactored a bit. Please refer to [this commit]([cebf260](https://github.com/Sitecore/jss/pull/1321/commits/cebf2603e542bfa8c6d7104c0f5de6c49870bea6)) to review the change.